packages/core/src/renderer/components/+cluster/cluster-overview.tsx

```diff
@@ -118,5 +121,6 @@ export const ClusterOverview = withInjectables<Dependencies>(NonInjectedClusterO
     podStore: di.inject(podStoreInjectable),
     eventStore: di.inject(eventStoreInjectable),
     nodeStore: di.inject(nodeStoreInjectable),
+    uiBlocks: di.injectMany(clusterOverviewUIBlockInjectionToken),
```

Comment:
Here we should use `computedInjectMany` to make it reactive. Currently it doesn't refresh if new implementation for  `clusterOverviewUIBlockInjectionToken` is registered in runtime. (or deregistered). 

This works at the moment because registration of the Feature is happening before anything is rendered, but it's good practice to always support runtime registrations (less work in the future when we enable possibility to register Features through runtime extensions)

Reply:
good call, fixed